### DISCUSSION TITLE
Add page margins option in the webtoon mode

### DISCUSSION THREAD

**Discussion by titan-19** [gotson/komga#1512] [Ideas]
### Describe your suggested feature

I'm simply talking about this option that exists in the suwayomi reader :
![éCapture](https://github.com/gotson/komga/assets/127576128/3016aa44-6143-431b-b054-c3b6e8e4132f)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
